CONTRIBUTORS.md

```diff
@@ -150,3 +150,5 @@ List of Contributors
 * [Manu Seth](https://github.com/mseth10/)
 * [Calum Leslie](https://github.com/calumleslie)
 * [Andre Tamm](https://github.com/andretamm)
+* [Marco de Abreu](https://github.com/marcoabreu)
+ - Marco is the current maintainer of the MXNet CI.
```

Comment:
I would suggest adding names of Meghna Baijal and Gautam Kumar as maintainers of the CI system.

Reply:
They're not part of this list yet and I don't know if I'm allowed to add them myself. It would probably be best if they that addition themselves.